src/fu-util.c

```diff
@@ -481,9 +481,13 @@ fu_util_download_if_required (FuUtilPrivate *priv, const gchar *perhapsfn, GErro
 static void
 fu_util_display_current_message (FuUtilPrivate *priv)
 {
-	if (priv->current_message == NULL)
+	if (priv->current_message == NULL) {
+		/* TRANSLATORS: success message */
+		g_print ("%s\n", _("Successfully installed firmware"));
 		return;
-	g_print ("%s\n", priv->current_message);
+	}
+	/* TRANSLATORS: success message with a custom message appended */
+	g_print ("%s%s\n", _("Successfully installed firmware: "), priv->current_message);
```

Comment:
It's unfortunate this turns into two translator strings.  I know you mentioned translators don't like you concatenating strings, but can this instead be:
`g_print ("%s: %s\n", _("Successfully installed firmware"), priv->current_message);`

for reuse

Reply:
Fixed